ccd-definition/UserProfile.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "UserIDAMId": "adoptionclerk@mailinator.com",
```

Comment:
just note if we want to use this new user in preview/etc we need to add it via IDAM WebAdmin